classes/class-wc-connect-shipping-method.php

```diff
@@ -446,15 +447,18 @@ public function calculate_shipping( $package = array() ) {
 							// Notify the merchant when the fallback rate is added by the WCS server.
 							$this->debug( 'No rates found, adding fallback.', 'error' );
 						} else {
-							$this->debug(
-								sprintf(
-									'Received rate: <strong>%s</strong> (%s)<br/><ul><li>%s</li></ul>',
-									$rate_to_add['label'],
-									wc_price( $rate->rate ),
-									implode( '</li><li>', $package_summaries )
-								),
-								'success'
+							$rate_debug  = '<strong>';
+							$rate_debug .= sprintf(
+								/* translators: 1: name of shipping service, 2: shipping rate (price) */
+								__( 'Received rate: %1$s (%2$s)', 'woocommerce-services' ),
+								$rate_to_add['label'],
+								wc_price( $rate->rate )
 							);
+							$rate_debug .= '</strong><ul><li>' . implode( '</li><li>', $package_summaries ) . '</li></ul>';
+							$rate_debug .= '<strong>' . __( 'Packing log:', 'woocommerce-services' ) . '</strong>';
+							$rate_debug .= '<ul><li>' . implode( '</li><li>', $rate->box_packing_log ) . '</li></ul>';
```

Comment:
When the box packing log does not exist, the packing log looks a bit weird. Possible to make sure it exists before displaying it?

<img width="826" alt="screen shot 2018-11-26 at 7 32 10 pm" src="https://user-images.githubusercontent.com/11487924/49050956-c6b52380-f1b3-11e8-916b-c69a0a51b5a4.png">

Reply:
Done in https://github.com/Automattic/woocommerce-services/pull/1542/commits/14166345792bea49b5184b13ee6af3fd901c9625.